### PR TITLE
Changes to README.md

### PR DESCRIPTION
**Pull request by LukeTheEngineer**
No changes to source code

### CODE REVIEW

**Comment by concussious** [comment]
As said in the previous PR, this is an awesome place to put this information.

My first attempt at PR here, I also closed the request and opened a new one. Several times actually. I think this caused clutter, confusion, and extra work for the grandmasters. Eventually I discovered, you can reopen the initial pr (even if the branch is deleted), rename the title, edit the description, anything, by playing with the buttons on the website. One continuous PR seems to make it easier for the comitter to include the history of the review into the tree.

Iirc comments on closed prs didn't send me an email.
https://github.com/freebsd/freebsd-src/pull/1140#issuecomment-2024480802

See this instruction to squash your commits into one with a great log message:
https://github.com/freebsd/freebsd-src/pull/825#issuecomment-1889670659

I (nobody, mind you, just a hobbyist here) recommend a pr and commit title like, "README.md: Mention image downloads", or "README.md: link bootable image downloads"... as long as those are <50 characters. I think it was CONTRIBUTING.md that has a great explanation for this.

**Comment by LukeTheEngineer** [comment]
Thanks for the knowledge

**Comment by bsdimp** [comment]
Can you squash all these down into one commit?

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ See build(7), config(8), [FreeBSD handbook on building userland](https://docs.fr
 For information on the CPU architectures and platforms supported by FreeBSD, see the [FreeBSD
 website's Platforms page](https://www.freebsd.org/platforms/).
 
+Official FreeBSD bootable images on our [releases page](https://download.freebsd.org/ftp/releases/ISO-IMAGES/) to get yourself one.
```
Comment (bsdimp) [inline README.md:18]: I don't think that the other parts of this file use 'our' or similar 1st-person plural pronouns.
`For official FreeBSD bootable images, see our [release page](...)`
might be better?

Comment (LukeTheEngineer) [reply README.md:18]: Is it an issue with using "our"?

Comment (bsdimp) [reply README.md:18]: Yes. It's an informal style that doesn't match the rest of the doc.

**Comment by LukeTheEngineer** [comment]
> Can you squash all these down into one commit?

How would I do that?

**Comment by concussious** [comment]
> > Can you squash all these down into one commit?
> 
> How would I do that?



> As said in the previous PR, this is an awesome place to put this information.
> 
> My first attempt at PR here, I also closed the request and opened a new one. Several times actually. I think this caused clutter, confusion, and extra work for the grandmasters. Eventually I discovered, you can reopen the initial pr (even if the branch is deleted), rename the title, edit the description, anything, by playing with the buttons on the website. One continuous PR seems to make it easier for the comitter to include the history of the review into the tree.
> 
> Iirc comments on closed prs didn't send me an email. [#1140 (comment)](https://github.com/freebsd/freebsd-src/pull/1140#issuecomment-2024480802)
> 
> See this instruction to squash your commits into one with a great log message: [#825 (comment)](https://github.com/freebsd/freebsd-src/pull/825#issuecomment-1889670659)
> 
> I (nobody, mind you, just a hobbyist here) recommend a pr and commit title like, "README.md: Mention image downloads", or "README.md: Link bootable image downloads"... as long as those are <50 characters. I think it was CONTRIBUTING.md that has a great explanation for this.

**Comment by bsdimp** [comment]
> > Can you squash all these down into one commit?
> 
> How would I do that?

`% git rebase -i main $BRANCH_NAME`

(where BRANCH_NAME is the name of your branch) then you'll have a number of lines in your editor. They will all be `pick <some-hex-number> Description of the commit`. Change all the 'pick' to fold (all after the first one). Save the file. This will tell git you want it to combine all the commits.

I don't think README.md has changed since your pull requests. If it had, you'd need to rebase by doing:
```
% git checkout main
% git pull
% git rebase -i main $BRANCH_NAME
```
(in this case, just leave what should just be one pick line). Hopefully you'd not have to resolve conflicts.

Once you've done all that, you'll need to push it back to GitHub with `% git push --force-with-lease`.

**Comment by LukeTheEngineer** [comment]
I've done git rebase -i main but the commits are no longer there. It displays noop

**Comment by bsdimp** [comment]
> I've done git rebase -i main but the commits are no longer there. It displays noop

Oh! You're not working on a branch. I hadn't noticed that. You'll need to change `main` to something like `origin/main` or `freebsd/main` depending on how you cloned this repo. `git remote -v` will tell you which one you'll likely want to use. `origin/main` is the default.

**Comment by bsdimp** [comment]
Landed... I think I closed it wrong though.

**Pull request closed without merging**